packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -54,7 +59,12 @@ const PersonalDetails = observer(
         const [should_close_tooltip, setShouldCloseTooltip] = React.useState(false);
         const [no_confirmation_needed, setNoConfirmationNeeded] = React.useState(false);
 
-        const PoiNameDobExampleIcon = PoiNameDobExample;
+        const side_note_image = (
+            <img
+                src={getUrlBase('/public/images/common/account/ic-poi-name-dob-example.png')}
```

Comment:
Nicely done @yauheni-deriv, but I do have a suggestion.
How about writing a hook e.g. useImage that accepts the name of the image and returns back the image? in that case you don't have to hardcode the path and the image name in multiple components.

```
const someComponent = () => {
  const imageName = 'ic-poi-name-dob-example.png';
  const image = useImage(imageName);

  return (
    <div>
      {image && <img src={image} alt="My Image" />}
    </div>
  );
};
```

Further than that, you can load the image `async` in your hook for a more optimised solution.

Reply:
Thanks @matin-deriv for your review.
I'm not sure I got your point.
I just used same approach as we use in deriv.app.
actually I had the idea to use  the function which returns us the path for image. 
we had this function in  bot package (getImageLocation).
We also can create the component, which will accept the name of the image and returns us the image.
Nut I guess we can just use <img> with function. 
let me create such function in shared and reuse it 🙏

Reply:
My point was more about using hooks and async functions for the best practices but your implementation is good too. I will just share the code with you.

```
import React, { useState, useEffect } from 'react';

const useImage = (imageName) => {
  const [image, setImage] = useState(null);

  useEffect(() => {
    const loadImage = async () => {
      try {
        const importedImage = await import(`/public/images/common/${imageName}`);
        setImage(importedImage.default);
      } catch (error) {
        console.error('Error loading image:', error);
      }
    };

    loadImage();

    return () => {
      // Clean up code if needed
    };
  }, [imageName]);

  return image;
};

export default useImage;
```

and 

```
const someComponent = () => {
  const imageName = 'ic-poi-name-dob-example.png';
  const image = useImage(imageName);

  return (
    <div>
      {image && <img src={image} alt="My Image" />}
    </div>
  );
};
```

Reply:
yeah, thank you!
I've seen such implementation.
I just trying to understand, why do we need to implement additional code (React hooks are not small functions) just to load 24kb from local folder. I believe it will have much more sense when we load the image from online resources. in this case we ahve to also add usestates for loading status and error.
how do you think? @matin-deriv